l10n_it_fatturapa_out_wt/wizard/wizard_export_fatturapa.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+from odoo import models
+from odoo.tools.translate import _
+from odoo.exceptions import Warning as UserError
+from odoo.addons.l10n_it_fatturapa.bindings.fatturapa_v_1_2 import (
+    DatiRitenutaType,
+    AltriDatiGestionaliType,
+)
+
+
+class WizardExportFatturapa(models.TransientModel):
+    _inherit = "wizard.export.fatturapa"
+
+    def setDatiGeneraliDocumento(self, invoice, body):
+        res = super(WizardExportFatturapa, self).setDatiGeneraliDocumento(
+            invoice, body)
+        ritenuta_lines = invoice.withholding_tax_line_ids.filtered(
+            lambda x: x.withholding_tax_id.wt_types == 'ritenuta')
+        if len(ritenuta_lines) > 1:
+            raise UserError(
+                _("More than one Ritenuta tax in invoice!"))
+        for wt_line in ritenuta_lines:
+            if not wt_line.withholding_tax_id.causale_pagamento_id.code:
+                raise UserError(_('Missing causale pagamento for wt %s!')
+                                % wt_line.withholding_tax_id.name)
+            body.DatiGenerali.DatiGeneraliDocumento.DatiRitenuta\
+                = DatiRitenutaType(
+                    TipoRitenuta="RT02" if invoice.partner_id.is_company
+                    else "RT01",  # RT02 persona giuridica
+                    ImportoRitenuta='%.2f' % wt_line.tax,
+                    AliquotaRitenuta='%.2f' % (wt_line.tax / wt_line.base),
```

Comment:
@sergiocorato questa parte credo sia errata. Io ho problemi con fattura con ritenuta 23% su 50% di imponibile, quindi l'aliquota dovrebbe essere del 11.50.  Con il tuo metodo di calcolo la ritenuta viene in decimali, quindi 0.115 e viene arrotondata a 2 decimali, quindi arrotondata a 0.12. 

Il codice dovrebbe essere (wt_line.tax / wt_line.base * 100). Allego preview di fattura elettronica da fornitore.

![image](https://user-images.githubusercontent.com/13367824/53720936-36494780-3e62-11e9-9ada-3811152a64f2.png)

Grazie, Giacomo